de/manual/ch_Business.docbook

```diff
@@ -1,1279 +1,1985 @@
+          Posten in einer Rechnung kann mit Hilfe der Steuertabellen unterschiedlich behandelt
+          werden.
         </para>
-      </note>
-
-      <para>You can specify the billing terms on each invoice/bill. Invoice billing terms will default from the
-        customer billing terms. Bill billing terms will default from the vendor billing terms.
-      </para>
-
-      <para>Billing Terms are maintained using the Billing Terms Editor which is accessed via menu
-        &mc.bus.bil-edit; . The following fields should be entered:
-      </para>
-
-      <itemizedlist>
-        <listitem>
-          <para><guilabel>Name</guilabel> The internal name of the billing term. For some examples of billing term
-            names and descriptions see <ulink url="&url-wiki-pmt-term;" />
-          </para>
-        </listitem>
-
-        <listitem>
-          <para><guilabel>Description</guilabel> The description of the billing term, printed on invoices
-          </para>
-        </listitem>
-
-        <listitem>
-          <para>There are 2 types of billing terms, with different information to be entered
-          </para>
+      </abstract>
+
+      <para>Mit Hilfe des Dialogs &mc.bus.tax-tab; können Sie die für Ihr Unternehmen notwendigen
+        Steuertabellen verwalten, anlegen und bearbeiten.
+      </para>
+
+      <figure pgwide="1">
+        <title>Steuertabellen verwalten</title>
+
+        <screenshot>
+          <screeninfo>820x240</screeninfo>
+
+          <mediaobject>
+            <imageobject>
+              <imagedata fileref="figures/Busnss_Tax_Organize.png" 
+                         srccredit="Christian Wehling" />
+            </imageobject>
+          </mediaobject>
+        </screenshot>
+      </figure>
+
+      <para>Der Dialog ist horizontal in zwei Bereiche unterteilt; unterhalb jedes Listenfeldes ermöglichen
+        Schaltflächen die Arbeit mit den darüber gewählten Listeneinträgen.
+      </para>
+
+      <glosslist>
+        <glossentry>
+          <glossterm>Steuertabellen</glossterm>
+          <glossdef>
+            <para>Die linke Liste enthält die unterschiedlichen Steuertabellen. Legen Sie hier die für Ihr
+              Unternehmen notwendigen Tabellen an.
+              <glosslist>
+                <glossentry>
+                  <glossterm>Umbenennen</glossterm>
+                  <glossdef>
+                    <para>Ermöglicht Ihnen die Vergabe eines neuen Names für die ausgewählte Steuertabelle.
+                    </para>
+                  </glossdef>
+                </glossentry>
+
+                <glossentry>
+                  <glossterm>Löschen</glossterm>
+                  <glossdef>
+                    <para>Löscht den markierten Eintrag aus der Liste.
+                      <warning>
+                        <para>Dieser Vorgang ist nicht reversibel! Erstellen Sie vorab eine Sicherungskopie Ihrer Datendatei.
```

Comment:
Ich wäre dafür statt Datendatei &app;-Buch zu verwenden.
Da bedarf eventuell einer Definition in den Konventionen.
Ich stelle mir da vor, die ersten 3 Kapitel zu einem [`<preface>`](https://tdg.docbook.org/tdg/4.5/preface) zu machen, etwa
Über …

1. Gnucash
2. Die Dokumentation
2.1. Teile
2.2. Konventionen 
3. Die Autoren

Ich könnte mir auch vorstellen, 1&2 als common in beide Dokumente zu linken.

Reply:
Ahh, eine neue Baustelle ;-)